doc/audit-log/Audit_Log_HLD.md

```diff
@@ -29,7 +29,7 @@ Audit log provides a way to monitor several security relevant information on the
 
 ## 1.1 Requirements
 
-- The log messages will be stored as a separate file under /var/log on the switch itself and will follow similar format to syslog messages. No support for remote logging or remote destination.
+- The log messages will be stored as a separate file under /var/log on the switch itself and will follow similar format to syslog messages.
```

Comment:
Need to add here that log messages will be also sent to remote logging server.

Reply:
Updated.